### PR TITLE
fix: Try another fix for cocoa test

### PR DESCRIPTION
**Pull request by wedamija**
Using `store_event` seems to help. The cocoa event from `load_data` doesn't seem to work with `store_event`, so just switched over to a python event. I think the choice of event type was arbitrary, so this shouldn't be a big problem

### CODE REVIEW

File: tests/acceptance/test_shared_issue.py [hunk 1]
```diff
@@ -21,23 +18,8 @@ def setUp(self):
...
-            first_seen=datetime(2015, 8, 13, 3, 8, 25, tzinfo=timezone.utc),
-            last_seen=datetime(2016, 1, 13, 3, 8, 25, tzinfo=timezone.utc),
-        )
-        return event
-
     def test_cocoa_event(self):
-        event = self.create_sample_event(
-            platform='cocoa',
-        )
+        event = create_sample_event(project=self.project, platform='cocoa')
```
Comment (markstory) [inline tests/acceptance/test_shared_issue.py]: You might want to fixate the event_id too so it doesn't get diffed in percy.

Comment (wedamija) [reply tests/acceptance/test_shared_issue.py]: Oh, is that why we hardcoded that... I don't see any differences in percy so far though?

**Comment by wedamija** [comment]
No luck, test is still unstable

**Comment by dashed** [comment]
Damb.

**Comment by wedamija** [comment]
This try hasn't failed, will try 2-3 more times

**Review by dashed — APPROVED** [review]
Ship it

**Pull request merged**